src/software/firmware/srcs/pressure_controller.cpp

```diff
@@ -434,12 +434,12 @@ PressureController::pidPatient(int32_t targetPressure, int32_t currentPressure,
     int32_t patientCommand = PID_PATIENT_KP * error + patientIntegral
                              + PID_PATIENT_KD * derivative / 1000;  // calcul de la commande
 
-    uint32_t patientAperture =
-        max(m_patient.minAperture(),
-            min(m_patient.maxAperture(),
-                m_patient.minAperture()
-                    + (patientCommand + 1000) * (m_patient.maxAperture() - m_patient.minAperture())
-                          / 2000));
+    uint32_t patientAperture = max(
+        m_patient.minAperture(),
+        min(m_patient.maxAperture(), m_patient.minAperture()
+                                         + (patientCommand + targetPressure)
```

Comment:
So if I understand correctly, this is not exactly a pid anymore since there is a term directly proportional to the target, not only to the error. You got better result with this formula, I guess.

Reply:
Indeed we get better results. 
Maybe we will improve this in a few days / hours according to the hardware modifications. New software modifications will be committed in a new PR.